src/scores/probability/crps_impl.py

```diff
@@ -219,35 +223,39 @@ def crps_cdf(
+        propagate_nans: If `True`, propagate NaN values along `threshold_dim` in `fcst` 
+            and `threshold_weight` prior to calculating CRPS. This will result in CRPS 
+            being NaN for these cases. If `False`, NaN values in `fcst` and `weight` will 
+            be replaced, wherever possible, with non-NaN values using the fill method 
             specified by `fcst_fill_method` and `threshold_weight_fill_method`.
-        fcst_fill_method: how to fill values in `fcst` when NaNs have been introduced
-            (by including additional thresholds) or are specified to be removed (by
-            setting `propagate_nans=False`). Select one of:
-                - "linear": use linear interpolation, then replace any leading or
-                  trailing NaNs using linear extrapolation. Afterwards, all values are
-                  clipped to the closed interval [0, 1].
-                - "step": apply forward filling, then replace any leading NaNs with 0.
-                - "forward": first apply forward filling, then remove any leading NaNs by
-                  back filling.
-                - "backward": first apply back filling, then remove any trailing NaNs by
-                  forward filling.
-            In most cases, "linear" is likely the appropriate choice.
+        fcst_fill_method: how to fill values in `fcst` when NaNs have been introduced 
+            (by including additional thresholds) or are specified to be removed (by 
+            setting `propagate_nans=False`). Select one of: 
+
+            - "linear": use linear interpolation, then replace any leading or \
+            trailing NaNs using linear extrapolation. Afterwards, all values are \
+            clipped to the closed interval [0, 1].
+            - "step": apply forward filling, then replace any leading NaNs with 0. 
+            - "forward": first apply forward filling, then remove any leading NaNs by \
+            back filling.
+            - "backward": first apply back filling, then remove any trailing NaNs by \
+            forward filling.
+            - (In most cases, "linear" is likely the appropriate choice.)
+
         threshold_weight_fill_method: how to fill values in `threshold_weight` when NaNs
             have been introduced (by including additional thresholds) or are specified
-            to be removed (by setting `propagate_nans=False`). Select one of "linear",
-            "step", "forward" or "backward". If the weight function is continuous,
-            "linear" is probably the best choice. If it is an increasing step function,
-            "forward" may be best.
+            to be removed (by setting `propagate_nans=False`):
+
+            - Select one of "linear", "step", "forward" or "backward". 
+            - If the weight function is continuous, "linear" is probably the best choice. 
+            - If it is an increasing step function, "forward" may be best.
```

Comment:
I'm convinced that adding dotpoints for these improves the readability of this paragraph. Obviously this is my subjective opinion.

Reply:
Can I just check - you are convinced, or you are not convinced? The original source code seemed to have a bulleted list, but I couldn't make sense of where the breaks between the bullet points were. I don't have any strong views about what this should be, I was just trying to tidy it up. I'm happy to remove the buillet points if that's what you are suggesting.

Reply:
oops - "not convinced". I was just suggesting that perhaps you should consider removing the bullet points (but it's not a strong opinion).